packages/lesswrong/components/notifications/NotificationsMenu.jsx

```diff
@@ -2,43 +2,59 @@ import { Components, registerComponent, withList } from 'meteor/vulcan:core';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Drawer from 'material-ui/Drawer';
-import Badge from 'material-ui/Badge';
+import Badge from '@material-ui/core/Badge';
 import {Tabs, Tab} from 'material-ui/Tabs';
 import AllIcon from '@material-ui/icons/Notifications';
 import ClearIcon from '@material-ui/icons/Clear';
 import PostsIcon from '@material-ui/icons/Description';
 import CommentsIcon from '@material-ui/icons/ModeComment';
 import MessagesIcon from '@material-ui/icons/Forum';
+import { withStyles } from '@material-ui/core/styles';
 import withUser from '../common/withUser';
+import classNames from 'classnames';
 
 // import { NavDropdown, MenuItem } from 'react-bootstrap';
 import Notifications from '../../lib/collections/notifications/collection.js'
 
-
-const badgeContainerStyle = {
-  padding: 'none',
-}
-const badgeStyle = {
-  backgroundColor: 'none',
-  color: 'rgba(0,0,0,0.6)',
-  fontFamily: 'freight-sans-pro, sans-serif',
-  right: "-16px"
-}
-
 const tabLabelStyle = {
   color: "rgba(0,0,0,0.8)",
   fontFamily: "freight-sans-pro, sans-serif"
 }
 
-const iconStyle = {
-  color: "rgba(0,0,0,0.8)",
-}
-
-const cancelStyle = {
-  color: "rgba(0,0,0,0.3)",
-  margin: "10px",
-  cursor: "pointer"
-}
+const styles = theme => ({
+  badgeContainer: {
+    padding: "none",
+    fontFamily: 'freight-sans-pro, sans-serif',
+    verticalAlign: "inherit",
+  },
+  badge: {
+    backgroundColor: 'inherit',
+    color: 'rgba(0,0,0,0.6)',
+    fontFamily: 'freight-sans-pro, sans-serif',
+    fontSize: "12px",
+    fontWeight: 500,
+    right: "-15px",
+    top: 0,
+    pointerEvents: "none",
+  },
+  menuContainer: {
+    height: "100vh",
+    boxShadow: "none",
+    transition: "transform 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
+    
+    // !important because material-UI defines an (incorrect)
+    // overflow
+    overflowX: "hidden !important"
+  },
+  icon: {
+    color: "rgba(0,0,0,0.8)",
```

Comment:
Don't have to fix this now, but note that we want to move these to grey values instead of opacity

Reply:
I'd like to do that everywhere at once, so I can figure out the correct gamma correction and blending for nonwhite backgrounds once.